<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsToolbars</class>
 <widget class="QWidget" name="SettingsToolbars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Button style</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbToolbarButtonStyle</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="m_cmbToolbarButtonStyle"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Icon size</string>
     </property>
     <property name="buddy">
      <cstring>m_spinToolbarIconSize</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="m_spinToolbarIconSize">
     <property name="minimum">
      <number>-1</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="singleStep">
      <number>8</number>
     </property>
     <property name="value">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Toolbar editor</string>
     </property>
     <property name="buddy">
      <cstring>m_cmbSelectToolBar</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="m_cmbSelectToolBar">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Toolbar for feeds list</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Toolbar for articles list</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Statusbar</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QStackedWidget" name="m_stackedToolbars">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_pageFeedsToolbar">
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ToolBarEditor" name="m_editorFeedsToolbar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pageMessagesToolbar">
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ToolBarEditor" name="m_editorMessagesToolbar" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_pageStatusbar">
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="ToolBarEditor" name="m_editorStatusbar" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolBarEditor</class>
   <extends>QWidget</extends>
   <header>toolbareditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_cmbToolbarButtonStyle</tabstop>
  <tabstop>m_spinToolbarIconSize</tabstop>
  <tabstop>m_cmbSelectToolBar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
